modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IO.kt

```diff
@@ -850,7 +784,7 @@ sealed class IO<out E, out A> : IOOf<E, A> {
    * @see [unsafeRunAsync] or [unsafeRunAsyncCancellable] that run the value as [Either].
    * @see [runAsync] to run in a referential transparent manner.
    */
-  fun unsafeRunSync(): A =
+  fun unsafeRunSync(): Either<E, A> =
```

Comment:
What happens with `IO<A>` then?

Reply:
Same reply as above. We need to think of a strategy for the API to distinct the 2 in an ergonomic way.